db/migrate/20171127171849_adds_common_id_to_endpoints.rb

```diff
@@ -0,0 +1,172 @@
+class AddsCommonIdToEndpoints < ActiveRecord::Migration
+  def change
+    execute <<-SQL
+    CREATE 
+    OR REPLACE VIEW "1"."user_details" AS       SELECT
```

Comment:
user_details was updated here https://github.com/catarse/catarse/commit/450750b617511b720cb1b90218cb9efde6e479a0

Reply:
removed it. thx! o/